assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -319,6 +319,7 @@ function Carousel() {
               <KeyboardShortcutsButton
                 width="24"
                 height="24"
+                isDisabled={true}
```

Comment:
This is just:
```suggestion
                isDisabled
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/975/commits/13691992cf61a8383f29d0ed470b0644954ee90b